content/questions/inceptionloop/index.md

```diff
@@ -17,9 +17,9 @@ const image = new Image;
 const maxHeight = image.height + 1;
```

Comment:
On line 16, the `Image` constructor expects `width` and `height` parameters, e.g. `new Image(300, 100);`